weaviate/collections/classes/config_named_vectors.py

```diff
@@ -196,6 +197,59 @@ def text2vec_cohere(
             vector_index_config=vector_index_config,
         )
 
+    @staticmethod
+    def multi2vec_cohere(
+        name: str,
+        *,
+        vector_index_config: Optional[_VectorIndexConfigCreate] = None,
+        vectorize_collection_name: bool = True,
+        base_url: Optional[AnyHttpUrl] = None,
+        model: Optional[Union[CohereModel, str]] = None,
+        truncate: Optional[CohereTruncation] = None,
+        image_fields: Optional[Union[List[str], List[Multi2VecField]]] = None,
+        text_fields: Optional[Union[List[str], List[Multi2VecField]]] = None,
+    ) -> _NamedVectorConfigCreate:
+        """Create a named vector using the `multi2vec_cohere` model.
+
+        See the [documentation](https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules/text2vec-cohere)
```

Comment:
Oh we need to update these docstrings - I will aim to add the docs for this tomorrow or so and add a link here.